app.py

```diff
@@ -52,7 +52,60 @@ def mirror(name):
     return create_response(data)
 
 
-# TODO: Implement the rest of the API here!
+@app.route("/users")
+def get_users():
+    data = db.get('users')
+
+    if "team" in request.args:
+        data = list(filter(lambda x: x["team"] == request.args["team"], data))
```

Comment:
Food for thought - what if I asked you to add more filter types? E.g. filter by name, etc? Can you think of a way to implement this filtering that's more generic / future proof?